include/dao/dao.js

```diff
@@ -114,6 +117,48 @@ module.exports = function DAOModule(pb) {
         this.loadByValues(where, collection, opts, cb);
     };
 
+    DAO.prototype.loadByValueAvailableToSite = function(key, val, site, collection, opts, cb) {
```

Comment:
@brianhyder is there anyway we can break out this refactor out of this PR? I would like to a bigger refactor that would touch more files and this PR is already crazy. It would be great if we could iterate.